pyaedt/generic/LoadAEDTFile.py

```diff
@@ -304,17 +307,16 @@ def _load_entire_aedt_file(filename):
         dictionary containing the decoded AEDT file
 
     """
-    _reaf_aedt_file(filename)
+    global _count
+    _read_aedt_file(filename)
+    main_dict = OrderedDict()
     # load the aedt file
-    begin_key = None
-    for line in _all_lines:
+    while _count < _len_all_lines:
+        line = _all_lines[_count]
         m = _begin_search.search(line)
         if m:
-            begin_key = m.group(1)
-            break
-    main_dict = OrderedDict()
-    if begin_key:
-        _walk_through_structure(begin_key, main_dict)
+            _walk_through_structure(m.group(1), main_dict)
+        _count = _count + 1
```

Comment:
This is a bit cleaner.  Performance gains are marginal on Python 3.8 (local), not sure on IronPython

```suggestion
        _count += 1
```